runtime/main.cpp

```diff
@@ -74,6 +77,18 @@ int main(int argc, char* const* argv) {
 	/// Shutting down method 1: Ctrl+C
     std::signal(SIGINT, sigint_handler);
 
+#ifndef NDEBUG
```

Comment:
A similar feature can be implemented by setting the `command` to:

```
gdb -quiet -ex run -args $cmd
```

Reply:
I think I have had issues starting ILLIXR under gdb with `xvfb` also enabled (issues with executable formats). I'll try this version.